src/index.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable no-useless-concat */
+/* eslint-disable prefer-template */
+
+import './style.css';
+import {
+  findMeals,
+  postComments,
+  postLikes,
+  likesUrl,
+} from './modules/getPostData.js';
+import {
+  displayPopUp,
+  showComments,
+  openPopup,
+  closePopup,
+  commentsContainer,
+} from './modules/userInterface.js';
+
+const mealsContainer = document.querySelector('#meal');
+const popupWindow = document.getElementById('pop-up');
+const form = document.querySelector('form');
+const input = document.getElementById('user-name');
+const textarea = document.getElementById('suggestion');
+const closeBtn = document.querySelector('.close-button');
+
+const displayMeals = async () => {
+  const meals = await findMeals();
+  meals.forEach((meal) => {
+    mealsContainer.innerHTML += `
+    <div id="${meal.idCategory}" class="meal-div">
+      <div class="image-cont">
+        <img
+          src=${meal.strCategoryThumb}
+          alt="food image"
+        />
+      </div>
+      <div>
+        <div><p>${meal.strCategory}</p></div>
+        <div class="home-btns">
+          <button class="comment-btn" type="button" id="${meal.idCategory}">Comments</button>
+          <i class="fa-solid fa-heart"><br><small id="${meal.idCategory}"class="likes-counter">0 likes</small></i>
+        </div>
+      
+  </div>
+  </div>
+        `;
+
+    // implement likes section
+    const likeBtns = document.querySelectorAll('.fa-heart');
+    const likesNo = document.querySelectorAll('.likes-counter');
+
+    likeBtns.forEach((i, index) => {
+      i.addEventListener('click', () => {
+        likesNo[index].innerHTML = parseInt(likesNo[index].innerHTML, 10) + 1 + `${' ' + 'likes'}`;
+        postLikes(index);
+      });
+    });
+  });
+  const getLikes = async () => {
```

Comment:
- [ ] The number of likes is not getting preserved in the API. When the page is refreshed it resets to zero. Please use [involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) to track the interaction of the user.